src/index.js

```diff
@@ -17,17 +17,17 @@ gisida.version = require('../package.json').version;
 gisida.initStore = require('./store/initStore').default;
 gisida.loadLayers = require('./store/initStore').loadLayers;
 gisida.reducerRegistry = require('./store/reducerRegistry').default;
-gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').default;
+gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').createMapReducer;
 
 gisida.reducers = {
-  ...app,
-  filter,
-  layersReducer,
-  loc,
-  locationsReducer,
-  regions,
-  styles,
-  supersetConfig,
+    ...app,
+    filter,
+    layersReducer,
+    loc,
+    locationsReducer,
+    regions,
+    styles,
+    supersetConfig,
```

Comment:
Let's keep tabbing at 2 spaces 🙏

Reply:
@kahummer for this and the other one that Conor noted below, can we add linting rule and [prettier](https://prettier.io/) (you can pair this nicely with [pre-commit](https://www.npmjs.com/package/pre-commit)) to assist us?